api/tests/tooling/database-builder/factory/build-user.js

```diff
@@ -24,7 +24,7 @@ const buildUser = function buildUser({
 } = {}) {
 
   password = _.isUndefined(password) ? encrypt.hashPasswordSync(faker.internet.password()) : encrypt.hashPasswordSync(password);
-  email = _.isUndefined(email) ? faker.internet.exampleEmail(firstName, lastName).toLowerCase() : email.toLowerCase();
+  email = _.isUndefined(email) ? faker.internet.exampleEmail(firstName, lastName).toLowerCase() : (null  !== email) ? email : null;
```

Comment:
add toLowerCase to email

Reply:
`email || null` ça fonctionnerait pas ici ?